app/src/main/java/it/ministerodellasalute/immuni/ui/countriesofinterest/CountriesListAdapter.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+ * Please refer to the AUTHORS file for more information.
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package it.ministerodellasalute.immuni.ui.countriesofinterest
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.widget.AppCompatCheckBox
+import androidx.recyclerview.widget.RecyclerView
+import it.ministerodellasalute.immuni.R
+import it.ministerodellasalute.immuni.extensions.nearby.ExposureNotificationManager.Companion.DAYS_OF_SELF_ISOLATION
+import it.ministerodellasalute.immuni.extensions.utils.DateUtils.MILLIS_IN_A_DAY
+import it.ministerodellasalute.immuni.extensions.view.setSafeOnClickListener
+import it.ministerodellasalute.immuni.logic.exposure.models.CountryOfInterest
+import java.util.*
+
+class CountriesListAdapter(private val clickListener: CountriesPreferences) :
+    RecyclerView.Adapter<CountriesListAdapter.CountriesVH>() {
+
+    var selectedCountries: MutableList<CountryOfInterest>? = mutableListOf()
```

Comment:
Since you initialise the list as an empty list you can remove the `?` and avoid using `!!` in the fragment.